project_1/part_1.py

```diff
@@ -118,17 +127,23 @@ def linear_regression(x, y, x_bar, y_bar):
         # intercept
         log_k = y_bar - b1*x_bar
 
-        return(log_k + (b1)*log_T)
+        return(log_k + (b1)*log_T, b1)
 
     # linear regression on log_B
-    linearized_log_B = linear_regression(log_T, log_B, mean_log_T, mean_log_B)
+    linearized_log_B, g_inv = linear_regression(log_T, log_B, mean_log_T, mean_log_B)
+
+    #linearized B
+    linearized_B = 10**(linearized_log_B)
 
     # plotting
-    plt.plot(log_T, linearized_log_B)
-    plt.plot(log_T, log_B)
-    plt.scatter(mean_log_T, mean_log_B)
+
+    plt.plot(log_T, linearized_log_B, label ='linearized estimate')
+    #plt.plot(log_T, log_B, label = 'observed brightness')
+    #plt.plot(T, linearized_B**(g_inv))
+    #plt.scatter(mean_log_T, mean_log_B))
     plt.xlabel('Logarithm of Exposure Time')
-    plt.ylabel('Logarithm of Brightness')
+    plt.ylabel('Linearized value of logarithm of Brightness')
+    plt.savefig('log T vs linearized log B for blue channel')
```

Comment:
Don't use space when naming a file in linux. It is usually better to use _  as it helps in tab completion when there are multiple files. So this would be log_T_vs_ .....